core/src/engine.cpp

```diff
@@ -40,6 +45,8 @@ void Engine::update_modules() {
     // 6) SYSTEM : Sorted Transparent Render Pass
     // 7) SYSTEM : Post Processing Stack
 
+    m_physicsModule->update_info_to_transform();
+
```

Comment:
this function should be called internally from inside of 'on_late_update();' on the physics subsystem